azure-functions/acbs-function/mappings/facility/facility-covenant.js

```diff
@@ -26,7 +25,7 @@ const facilityCovenant = (deal, facility, covenantType) => {
     facilityIdentifier: facility.facilitySnapshot.ukefFacilityId.padStart(10, 0),
     portfolioIdentifier: CONSTANTS.FACILITY.PORTFOLIO.E1,
     covenantType,
-    maximumLiability: helpers.getMaximumLiability(facility.facilitySnapshot),
+    maximumLiability: Number(facility.facilitySnapshot.value),
```

Comment:
is `helpers.getMaximumLiability` no longer used or needed now?

Reply:
No longer needed due to AC.

Reply:
Just wondering if it's worth keeping the helper function or if we should remove? Happy either way 👍